app/ViewModels/ViewModelFactory.php

```diff
@@ -44,6 +44,7 @@ public static function collection(Collection $models): Collection
 
     public static function paginate(LengthAwarePaginator $paginator): LengthAwarePaginator
     {
+        // @phpstan-ignore-next-line
         $paginator->getCollection()->transform(fn ($model) => static::make($model));
```

Comment:
The static analysis fails because we typehint `LengthAwarePaginator` interface that does not contain the `getCollection` method, but it does. We can change the argument type to be the concrete implementation `Illuminate\Pagination\LengthAwarePaginator`, but that seems as a too much of a gamble as we use the `ViewModelFactory::paginate()` method all over the place.